abstract-factory/typescript/command.ts

```diff
@@ -0,0 +1,3 @@
+export abstract class Command {
+  public abstract executeCommand(query: string): Promise<void>
```

Comment:
Buralarda `Promise<void>`  dönmeye gerek var mıydı acaba, kalmasının bir mahsuru yok ama sadece aklıma takıldı.

Reply:
Asenkron olması muhtemel yerlerde promise için de bir örnek bulunsun diye düşündüm fakat daha kafa karıştırıcı olabilir gibi. Promise olan yerleri kaldırıyorum 👍